### DISCUSSION TITLE
Change kong error into a static html file

### DISCUSSION THREAD

**Discussion by mrzdtydlntm** [Kong/kong#11400] [Help] [answered]
Hi!
I want to ask a question. I'm using kong inside a docker compose. In my kong app, I'm just using route /general and /auth, and when I access the root route (/) it returns "kong error no route matched". Can I change the kong error page into a static file html?

Thank you for helping :)

**Comment by shettyh** [comment, marked as answer]
Check if this helps https://docs.konghq.com/gateway/latest/reference/configuration/#error_template_html

**Reply by mrzdtydlntm** [reply]
Thank you so much. It works at me. Next i want to check if the kong can detect the upstream connection. If the port is not accessible, i want to serve 502 static html page!
